historical/views/containers/chatroom/MessageInteractive.vue

```diff
@@ -0,0 +1,205 @@
+<template lang='pug'>
```

Comment:
Saved this file inside `history` folder, because I am not sure we could use this file or content later.